android/src/main/java/com/clevertap/react/CleverTapApplication.kt

```diff
@@ -1,9 +1,11 @@
 package com.clevertap.react
 
+import com.clevertap.android.sdk.ActivityLifecycleCallback
 import com.clevertap.android.sdk.Application
 
 open class CleverTapApplication : Application() {
     override fun onCreate() {
+        ActivityLifecycleCallback.register(this)
```

Comment:
https://developer.clevertap.com/docs/react-native-quick-start-guide#setup-androids-application-class

We ask the client's application class to register the activity lifecycle. This way it will be done twice if they extend the ClevertapApplication class

Could that led to some unknown issues?

Reply:
```ActivityLifecycleCallback.register``` does nothing if it has already been registered. I didn't see a reason not to include in this class, since it is suppose to ease the integration. Also in the docs in integration.md 4. it stated that clients can either extend CleverTapApplication or register the callbacks manually, but CleverTapApplication didn't registered them.

Reply:
Makes sense as it doesn't break the current integration. Several clients extend CleverTapApplication and might also be calling register